QMigrate/lut/create_lut.py

```diff
@@ -353,7 +357,6 @@ def _compute_1d_sweep(lut, phase, vmodel, **kwargs):
     grid_xyz = [g / km_cf for g in lut.grid_xyz]
     stations_xyz = lut.stations_xyz / km_cf
     ll, *_, ur = lut.grid_corners / km_cf
-    nlloc_dx /= km_cf
     vmodel = vmodel / km_cf
```

Comment:
I think everything downstream of this should also have the phase velocity units in the docstrings (km/s), as you’ve done for station_xyz etc.

Reply:
So `_vmodel_string`, `_velocity_gradient`, `_grid_string`, `_write_control_file`?